wiki/Beatmap/Overall_difficulty/en.md

```diff
@@ -30,6 +30,10 @@ Hit windows for judgments can be viewed when hovering over the [beatmap informat
 | 100 | `140 - 8 * OD` |
 | 50 | `200 - 10 * OD` |
 
+#### Table comparison
```

Comment:
seems unnecessary, table can go in same section as above

Reply:
i don't have too much of an opinion, but if it's removed then the same should be done in the ar article

Reply:
okay